src/content/docs/en/pages/secure-journey/intelligent-dns/lets-encrypt-record.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: How to add a TXT record to configure Let's Encrypt certificate
```

Comment:
To achando que devemos pular Let's Encrypt pra esse caso, porque o TXT não precisa ser criado manualmente...

Olha o que o gepeto sugeriu:

```
Use Case: Domain Ownership Verification for Webmaster Tools

Many webmaster tools and services, such as Google Search Console, require website owners to verify that they have control over the domain. To do this, they often provide a unique token or code that the website owner must add to their domain's DNS settings.

In this case, a TXT record can be used to store this verification token. The website owner would add a TXT record to their DNS configuration with the provided token. The DNS information is then queried by the webmaster tools service to confirm that the website owner has control over the domain.

For example, if Google Search Console provides a unique token like "google-site-verification=abcdefgh12345678," the website owner would add a TXT record like this:

Name: @
Type: TXT
Value: "google-site-verification=abcdefgh12345678"

This indicates to Google that the website owner has added the required verification token to the domain's DNS settings and therefore has control over the domain.

It's worth noting that TXT records have various other applications, including storing SPF (Sender Policy Framework) records for email authentication and providing additional information for various services. The content of a TXT record can be plain text or formatted data depending on the specific use case.
```